xml/System.ComponentModel.Composition/MetadataViewImplementationAttribute.xml

```diff
@@ -50,7 +50,13 @@
       <Docs>
         <param name="implementationType">The type of the metadata view.</param>
         <summary>Initializes a new instance of the <see cref="T:System.ComponentModel.Composition.MetadataViewImplementationAttribute" /> class.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+By default MetadataViews are generated using reflection emit.  This attribute allows the developer to specify the ttype that implements the view rather than using a generated type.
```

Comment:
```suggestion
By default, MetadataViews are generated using reflection emit.  This attribute allows the developer to specify the type that implements the view rather than using a generated type.
```